planning/behavior_velocity_planner/include/scene_module/intersection/scene_intersection.hpp

```diff
@@ -111,8 +112,12 @@ class IntersectionModule : public SceneModuleInterface
     double intersection_velocity;  //! used for intersection passing time
     double intersection_max_acc;   //! used for calculating intersection velocity
     double detection_area_margin;  //! used for detecting objects in detection area
```

Comment:
Is this variable still used?

Reply:
use it here
https://github.com/autowarefoundation/autoware.universe/blob/4c910acd059e566c433ff3fd9bb9d83e1daf1e64/planning/behavior_velocity_planner/src/scene_module/intersection/scene_intersection.cpp#L608